.github/workflows/pr-node-tests.yml

```diff
@@ -15,7 +15,8 @@ jobs:
         node-version:
           - '14'
         leia-tests:
-          - node-10-example
```

Comment:
Spoke with Pirog and he wants us to keep all the legacy version options available/tested. Let's keep the node-10-example tests @badrange

Reply:
Ah, makes sense! I pushed an update, let's hope the CI/CD pipeline works.